tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -477,7 +477,7 @@ public void EnableUnixDomainSockets()
 #endif
         }
 
-        public MockTracerAgent GetMockAgent(bool useStatsD = false)
+        public MockTracerAgent GetMockAgent(bool useStatsD = false, int fixedPort = -1)
```

Comment:
Using a fixed port often makes for flaky tests if the port isn't available - is this necessary, or can we work around it?

Reply:
yes I think this is necessary, see my other comment